bench-vortex/src/reader.rs

```diff
@@ -47,6 +47,17 @@ pub fn open_vortex(path: &Path) -> VortexResult<Array> {
         .map(|a| a.into_array())
 }
 
+pub async fn open_vortex_async(path: &Path) -> VortexResult<Array> {
```

Comment:
this is just copy-paste but without the block_on

Reply:
I really wish there was a way not to fork these functions... but I think this is fixed once we have some higher level abstractions. Or benchmarks are just tokio.